examples/image_classification-tf.ipynb

```diff
@@ -0,0 +1,7883 @@
+{
```

Comment:
<p>Would be interesting to benchmark the performance of this pipeline and a pure <code>tf.data</code> pipeline (I understand they might overlap here). </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
Cc @Rocketknight1

Reply:
<p>Comparing pipelines is definitely interesting and something to think about :) I'm not sure it belongs here as these notebooks are demoing a pipeline of how to do X and not necessarily comparing. </p>

Reply:
I'm sorry -- I should have been clearer. I just left that comment as a thought. The notebook absolutely doesn't need to include that.

Reply:
<p>I suspect pure tf.data will be a lot faster right now, but we're working on that!</p>

Reply:
I would love to contribute to it if possible. Are there open parts there?